src/oncoexporter/cda/mapper/op_uberon_mapper.py

```diff
@@ -1,13 +1,13 @@
-import os
+from typing import Optional
 
 from .op_mapper import OpMapper
-from typing import Optional
 import pandas as pd
-from collections import defaultdict
 import phenopackets as PPkt
 
+
 class OpUberonMapper(OpMapper):
```

Comment:
Eventually we should rename these "Op*" variables and class names. Op I think used to mean oncopackets, but now is just confusing

Reply:
Yes, it is a bit confusing. We should rename when we're ready for the change.

Reply:
obviously this is not in scope for this PR, just thought I would mention this while I was reviewing the code